client/src/components/pages/home/Home.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from "react";
+import TourneyCard from "./TourneyCard";
+import SocialCard from "./SocialCard";
+import "./Home.css";
+
+import data from "../../../content/home-en";
+import socials from "../../../content/socials";
+import { Col, Layout, Row } from "antd";
+import PastTournies from "./PastTournies";
+import { useMatchMedia } from "../../../utilities";
+import HomeBanner from "./HomeBanner";
+const { Content } = Layout;
+
+function organizeIntoGrid(tournies, columns = 2) {
+  var tourneyGrid = [];
+
+  for (var i = 0; i < tournies.length; i += columns) {
+    var row = tournies.slice(i, i + columns);
+    tourneyGrid.push(row);
+  }
+
+  return tourneyGrid;
+}
+
+/**
+ * @param {import("./HomeBanner").HomeBannerProps} props
+ */
+export default function Home({ user, setUser }) {
+  useEffect(() => {
+    document.title = "GTS";
+  });
+
+  var mobileLayout = useMatchMedia("(max-width: 1280px)")?.matches;
```

Comment:
nit: prefer const

Reply:
Fixed in 400d747fcda341fb788f49c155bcf464530e1af2